newton/tests/test_menagerie_usd_mujoco.py

```
@@ -0,0 +1,1577 @@
+        3: "tendon",  # mjTRN_TENDON
+        4: "site",  # mjTRN_SITE
+        5: "body",  # mjTRN_BODY
+    }
+    trntype = int(mj_model.actuator_trntype[act_idx])
+    trnid = int(mj_model.actuator_trnid[act_idx, 0])
+    resolver = _TRN_RESOLVERS.get(trntype)
+    if resolver is not None and hasattr(mj_model, resolver):
+        try:
+            return getattr(mj_model, resolver)(trnid).name
+        except Exception:
+            pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)